src/validation.cpp

```diff
@@ -1238,12 +1238,10 @@ CChainState::CChainState(CTxMemPool* mempool,
                          ChainstateManager& chainman,
                          CEvoDB& evoDb,
                          const std::unique_ptr<CChainstateHelper>& chain_helper,
-                         const std::unique_ptr<llmq::CChainLocksHandler>& clhandler,
```

Comment:
is it possible to drop `#include <llmq/chainlocks.h>` from `validation.cpp`?
Is it possible to drop `class CChainLocksHandler;` from `validation.h`?

Reply:
same for `class CInstantSendManager` and `llmq/instantsend.h`

Reply:
* `llmq::CChainLocksHandler` is still referenced by `TestBlockValidity` ([source](https://github.com/dashpay/dash/blob/f7d9212f1510180367b58f0232700eff2e2f3c20/src/validation.h#L353))
* `llmq::CInstantSendManager` has been dropped from `validation.cpp` in latest push